core/src/main/java/org/geysermc/geyser/util/Metrics.java

```diff
@@ -48,9 +49,10 @@
 
 /**
  * bStats collects some data for plugin authors.
- *
- * Check out https://bStats.org/ to learn more about bStats!
+ * <p>
+ * Check out <a href="https://bStats.org/">...</a> to learn more about bStats!
  */
+@SuppressWarnings("unused")
 public class Metrics {
```

Comment:
I would just remove everything that isn't used

If at some point we need something we removed, it would be best to check the most up to date class on Metrics' github. Especially considering the majority of this class hasn't changed in 4 years lol